docs/post-processors/vsphere.mdx

```diff
@@ -69,6 +69,13 @@ Optional:
 - `overwrite` (boolean) - If it's true force the system to overwrite the
   existing files instead create new ones. Default is `false`
 
+- `hardware_version` (string) - The virtual hardware version that the VM
+  should be deployed with. Use when deploying to vsphere/esxi host whose version
+  is different than the vsphere/esxi host used to create the artifact. See
+  [VMWare's virtual hardware KB article]
```

Comment:
"VMware"

Reply:
Ah ! I didn't see that one ! Feel free to open a new PR to fix it :)

Reply:
Thanks // sorry ! :)

Reply:
Addressed these and others concerns in https://github.com/hashicorp/packer-plugin-vsphere/pull/153.

Ryan

Reply:
Dope !